src/main/resources/application-test.properties

```diff
@@ -0,0 +1 @@
+
```

Comment:
I prefer to keep the template as minimal as possible so that whoever wants to use it, they can customize it as per their needs and preferences. As of now there are no "test" profile specific properties, so I think this file is not required yet.